RFC/src/RFC-0140 Syncing and seeding.md

```diff
@@ -0,0 +1,131 @@
+# RFC-0140/Sync and Seeding
```

Comment:
Following C4, This should be a one-word short name matching the filename with no spaces. 

```suggestion
# RFC-0140/SyncAndSeeding
```